OpenSim/ExampleComponents/HopperDevice.h

```diff
@@ -96,9 +103,9 @@ class OSIMEXAMPLECOMPONENTS_API HopperDevice : public ModelComponent {
         //       to a bug where PrescribedController will only control the
         //       first actuator found with a given name. Re-visit once
         //       PrescribedController issue is resolved.
```

Comment:
Should either remove this TODO or change it (with some explanation that the `clutchedPathSpring` draws its own color).